dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -460,6 +460,10 @@ info.config.conflict=CWWKG0102I: {0}
 info.config.conflict.explanation=Multiple values are specified for the same configuration property. 
 info.config.conflict.useraction=No action is required. The informational message can be avoided by editing the server configuration to remove the conflict.
  
+warning.supplied.config.not.valid=CWWKG0103W: The attribute name {0} with value {1} will be ignored because it is defined as ibm:final. The system will use the default value of the attribute.
+warning.supplied.config.not.valid.explanation= Supplying a configuration attribute marked as ibm:final and default is not valid.  Remove the configured attribute from the configuration.
```

Comment:
What about, "A configuration attribute marked as ibm:final and default is not valid."?

Also, we only need one space after a period that ends a sentence.

Reply:
Hey @sishida .  Will take your comments into account once we determine a final resolution.  Having to take this one to a Design issues mtg as it may impact customers given what we've initially coded.  Will let you know if the msgs change and need a further/final review.